packages/client/lib/client.ts

```diff
@@ -120,6 +122,14 @@ export class EthereumClient {
 
     await Promise.all(this.services.map((s) => s.open()))
 
+    const ts = this.chain.genesis.header.timestamp
+    const tsDiff = Number(ts) - Date.now()
```

Comment:
```suggestion
    const tsDiff = Number(ts) - Date.now()/1000
```
`Date.now() is in milliseconds`